README.md

```diff
@@ -1,4 +1,4 @@
-Rocket Chip Generator :rocket: [![Build Status](https://travis-ci.org/chipsalliance/rocket-chip.svg?branch=master)](https://travis-ci.org/chipsalliance/rocket-chip)
+Rocket Chip Generator :rocket: ![Build Status](https://github.com/chipsalliance/rocket-chip/workflows/Continuous%20Integration/badge.svg?branch=master)
```

Comment:
Note: This removes the hyperlink that wrapped the image and that pointed to the latest build page, since there does not appear to be a way to get a static URL to the "latest" run of a workflow in GitHub Actions.